powerhome/playbook
# Expose popover's minWidth prop for Filter Kit.

#### Breaking Changes
No. Just added additional prop (`minWidth`) to the `Filter` kit. 

#### Runway Ticket URL
https://github.com/powerhome/playbook/issues/1232


#### How to test this
Add a minWidth to the Filter kit as a prop and see it change the size of the popover.

#### Checklist:

~- [ ] **SPECS** Please cover your changes with specs.~

*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
